src/connection/manager.ts

```diff
@@ -117,10 +117,13 @@ export class SQLJobManager {
     const selected = this.jobs[this.selectedJob]
     if (ServerComponent.isInstalled() && selected) {
       return selected.job.query<T>(query, opts);
-    } else if(!ServerComponent.isInstalled()) {
+    } else if (!ServerComponent.isInstalled()) {
       let updateResult = await ServerComponent.checkForUpdate();
-      if(UpdateStatus.JUST_UPDATED === updateResult) {
-        setTimeout(() => {JobManagerView.setVisible(true)});
+      if (UpdateStatus.JUST_UPDATED === updateResult) {
+        JobManagerView.setVisible(true);
+        if (`new`=== Configuration.get<string>(`alwaysStartSQLJob`)) {
+          await commands.executeCommand(`vscode-db2i.jobManager.newJob`);
```

Comment:
When I pulled down the updates, my VS Code wasn't happy about these lines.
![image](https://github.com/halcyon-tech/vscode-db2i/assets/134640655/db42551a-de37-4380-b3f9-7c98e29c8069)

I needed to add imports to satisfy it.
``` typescript
import { commands } from "vscode";
import Configuration from "../configuration";
```

Reply:
sorry
https://github.com/halcyon-tech/vscode-db2i/pull/74/commits/685feb475eee0eaf9169f9bf555c64df3d6d5f29